llvm/lib/CodeGen/RegAllocGreedy.cpp

```diff
@@ -135,6 +136,10 @@ static cl::opt<bool> GreedyRegClassPriorityTrumpsGlobalness(
              "more important then whether the range is global"),
     cl::Hidden);
 
+static cl::opt<std::string> TrainingLog(
```

Comment:
nit: rename the variable to PriorityTrainingLog - too many training logs everywhere :)